package.json

```diff
@@ -4,20 +4,25 @@
     "private": true,
     "description": "ReefGuide mono repo",
     "scripts": {
+        "prebuild": "pnpm run generate",
+        "predev": "pnpm run generate",
```

Comment:
seems like this would fix prisma types build, but it doesn't. (tested `npm run dev` in root)

I'll remove these since the `"prebuild": "prisma generate"` line I added in db's `package.json` works